db/schema.rb

```diff
@@ -175,6 +160,21 @@
 
   add_index "roles", ["name"], name: "roles_name_idx", unique: true, using: :btree
 
+  create_table "steps", force: :cascade do |t|
+    t.integer  "user_id"
```

Comment:
since this table is now not approval-specific, is this the time to also refactor it to use `proposal_roles`? I.e. how do we tell by looking at a `step` what flavor it is?